packages/host/app/components/operator-mode/submode-layout.gts

```diff
@@ -165,33 +167,35 @@ export default class SubmodeLayout extends Component<Signature> {
           @defaultLengthFraction={{1}}
           @minLengthPx={{371}}
           @collapsible={{false}}
+          class='main-panel'
         >
           <SubmodeSwitcher
             @submode={{this.operatorModeStateService.state.submode}}
             @onSubmodeSelect={{this.updateSubmode}}
             class='submode-switcher'
           />
           {{yield this.openSearchSheetToPrompt}}
+          <AiAssistantButton
+            class='chat-btn'
+            @isActive={{this.isAiAssistantVisible}}
+            {{on 'click' this.toggleChat}}
+          />
```

Comment:
This change means the button shows even when the AI bot isn’t running, it just does nothing, which seems incorrect to me.

![screencast 2024-03-20 15-52-01](https://github.com/cardstack/boxel/assets/43280/0a4466bb-9ef7-4eeb-8c66-3a1610033c8b)

Reply:
Whoops -- good catch